.github/workflows/automerge.yml

```diff
@@ -1,36 +0,0 @@
-name: automerge
-on:
-  pull_request:
-    types:
-      - labeled
-      - unlabeled
-      - synchronize
-      - opened
-      - edited
-      - ready_for_review
-      - reopened
-      - unlocked
-  pull_request_review:
-    types:
-      - submitted
-  check_suite:
-    types:
-      - completed
```

Comment:
We might be able to remove this, so that it does not trigger on other checks?

Reply:
I guess automerge just won't run when CI complete if we do that, making it useless.